kombu/async/aws/sqs/message.py

```diff
@@ -2,45 +2,32 @@
 """Amazon SQS message implementation."""
 from __future__ import absolute_import, unicode_literals
 
-from .ext import (
-    RawMessage, Message, MHMessage, EncodedMHMessage, JSONMessage,
-)
+from kombu.message import Message
+import base64
 
-__all__ = [
-    'BaseAsyncMessage', 'AsyncRawMessage', 'AsyncMessage',
-    'AsyncMHMessage', 'AsyncEncodedMHMessage', 'AsyncJSONMessage',
-]
 
-
-class BaseAsyncMessage(object):
+class BaseAsyncMessage(Message):
     """Base class for messages received on async client."""
 
-    def delete(self, callback=None):
-        if self.queue:
-            return self.queue.delete_message(self, callback)
-
-    def change_visibility(self, visibility_timeout, callback=None):
-        if self.queue:
-            return self.queue.connection.change_message_visibility(
-                self.queue, self.receipt_handle, visibility_timeout, callback,
-            )
 
-
-class AsyncRawMessage(BaseAsyncMessage, RawMessage):
+class AsyncRawMessage(BaseAsyncMessage):
     """Raw Message."""
 
 
-class AsyncMessage(BaseAsyncMessage, Message):
+class AsyncMessage(BaseAsyncMessage):
     """Serialized message."""
 
-
-class AsyncMHMessage(BaseAsyncMessage, MHMessage):
-    """MHM Message (uhm, look that up later)."""
-
-
-class AsyncEncodedMHMessage(BaseAsyncMessage, EncodedMHMessage):
-    """Encoded MH Message."""
-
-
-class AsyncJSONMessage(BaseAsyncMessage, JSONMessage):
-    """Json serialized message."""
+    def encode(self, value):
+        """Encode/decode the value using Base64 encoding."""
+        return base64.b64encode(value).decode('utf-8')
```

Comment:
I've since learned that encoding to Base64 is recommend at it appears that Boto3 does not do this automatically. Deleting the above comment.